samwise.js

```diff
@@ -1,37 +1,36 @@
 /**
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.